src/components/Users/Users.js

```diff
@@ -1,69 +1,71 @@
 import React from 'react';
 
-// import * as axios from 'axios';
+import * as axios from 'axios';
 
-// axios.get('https://social-network.samuraijs.com/api/1.0/users').then(response => {
-//     props.setUsers()
-// });
+import userPhoto from '../../assets/images/unnamed.jpg'
 
 import styles from './Users.module.css';
 
-const Users = (props) => {
-  const span1 = (user) => {
-    const { photoUrl, id, followed } = user;
+class Users extends React.Component {
+    constructor(props) {
+        super(props);
+        axios.get('https://social-network.samuraijs.com/api/1.0/users')
+            .then(response => {
+                this.props.setUsers(response.data.items);
+                console.log(response)
+            });
+    };
 
-    return (
-      <span>
-        <div>
-          <img src={photoUrl} className={styles.Photo} />
-        </div>
-        <div>
-          {followed
-            ? <button onClick={() => { props.unfollow(id); }}>Unfollow</button>
-            : <button onClick={() => { props.follow(id); }}>Follow</button>}
-        </div>
-      </span>
-    );
-  };
+    goMap = () => {
+        const newMap = this.props.users.map((user) => (
+            <div key={user.id}>
+                {this.span1(user)}
+                {this.span2(user)}
+            </div>
+        ));
 
-  const span2 = (user) => {
-    const { fullName, status } = user;
-    const { country, city } = user.location;
+        return newMap
+    };
 
-    return (
-      <span>
-        <span>
-          <div>
-            {fullName}
-          </div>
-          <div>
-            {status}
-          </div>
-        </span>
-        <span>
-          <div>
-            {country}
-          </div>
-          <div>
-            {city}
-          </div>
-        </span>
-      </span>
-    );
-  };
+    span1 = (user) => {
 
-  const newMap = props.users.map((user) => (
-    <div key={user.id}>
-      {span1(user)}
-      {span2(user)}
-    </div>
-  ));
+        return (
+            <span>
+                <div>
+                    <img src={user.photos.small != null ? user.photos.small : { userPhoto }} className={styles.Photo} />
+                </div>
+                <div>
+                    {user.followed
+                        ? <button onClick={() => { this.props.unfollow(user.id); }}>Unfollow</button>
+                        : <button onClick={() => { this.props.follow(user.id); }}>Follow</button>}
+                </div>
+            </span>
+        );
+    };
 
-  return (
-    <div>
-      {newMap}
-    </div>
-  );
+    span2 = (user) => {
+
+        return (
+            <span>
+                <span>
+                    <div>
+                        {user.name}
+                    </div>
+                    <div>
+                        {user.status}
+                    </div>
+                </span>
+            </span>
+        );
+    };
+
+    render() {
+        return (
+            <div>
```

Comment:
Is this div necessary?
If not, consider using Fragment.

Reply:
This will be possible with the definition of styles